docs/pages/blog/material-ui-v6-is-out.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Material UI v6 is out 🎉
+description: We are so excited to announce the stable release of Material UI v6. It comes with Pigment CSS integration and a lot of improvements.
+date: 2024-08-09T00:00:00.000Z
+authors:
+  [
+    'aarongarciah',
+    'brijeshb42',
+    'diegoandai',
+    'mnajdova',
+    'samuelsycamore',
+    'siriwatknp',
+    'zanivan',
+  ]
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+Material UI v6 is now stable! We're excited to share all the updates.
+
+## Table of contents
+
+- [New features](#new-features)
+  - [CSS theme variables](#css-theme-variables)
+  - [Color schemes](#color-schemes)
+  - [CSS media `prefers-color-scheme`](#css-media-prefers-color-scheme)
+  - [Container Queries](#container-queries)
+  - [New method of applying styles](#new-method-of-applying-styles)
+- [Improvements](#improvements)
+  - [Revamping the free templates](#revamping-the-free-templates)
+  - [Stabilized Grid v2](#stabilized-grid-v2)
+- [Experimental CSS extraction via Pigment CSS](#experimental-css-extraction-via-pigmentcss)
+  - [React Server Components](#react-server-components)
+  - [Built-in sx prop support](#built-in-sx-prop-support)
+- [Package size reduction](#package-size-reduction)
```

Comment:
Could "Package size reduction" be places under "Improvements"?

Reply:
👍 Ah, yes. That's better.